topics/atom/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: atom-package, atom-packages, atom-theme, atom-themes
```

Comment:
I think these would make more sense in the `related` field instead of `aliases`. They have different meanings than "atom" but are definitely associated.

Reply:
Actually, sorry about that, I think these are fine here. Keeping them as aliases should let this atom content show up on the topic pages for atom-package, etc.